app/templates/buyers/brief_overview.html

```diff
@@ -47,7 +47,7 @@
               heading = "Are you sure you want to withdraw these requirements?",
               banner_content = "<p>{}</p><ul><li>{}</li></ul>".format(withdraw_text, withdraw_list_items|join("</li><li>"))|safe,
               type = "destructive",
-              action = '<button type="submit" class="button-destructive banner-action">Withdraw requirements</button>'|safe
+              action = '<button class="govuk-button govuk-button--warning app-banner-action" data-module="govuk-button">Withdraw requirements</button>'|safe
```

Comment:
Can't we call the button component here instead of having a hardcoded button?

Reply:
Yeah I guess - was just trying to make it as much a 1:1 translation as possible.